server/@types/models/Referral.ts

```diff
@@ -18,4 +18,6 @@ type ReferralUpdate = {
   reason?: string
 }
 
-export type { CreatedReferralResponse, Referral, ReferralUpdate }
+type ReferralUpdateStatus = 'awaiting_assesment' | 'assessment_started' | 'referral_started' | 'referral_submitted'
```

Comment:
Is this not just a `ReferralStatus`? I think we should be able to reuse this later when we need to read the status?

Reply:
Good shout!